### PR TITLE
Initialize the Notifier

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

**Comment by roj1512** [comment]
Please let me know if we need to have any text talking about the bot and why is it existing.

File: notifier/mod.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { Bot } from "https://deno.land/x/grammy/mod.ts";
+import { Application } from "https://deno.land/x/oak/mod.ts";
```
Comment (KnorpelSenf) [inline notifier/mod.ts:2]: No versions?

File: notifier/env.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import "https://deno.land/x/dotenv/load.ts";
+import { cleanEnv, num, str } from "https://deno.land/x/envalid/mod.ts";
```
Comment (KnorpelSenf) [inline notifier/env.ts:2]: No versions?

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
I would add a short README file, just 3 lines or so, and a link to the translation guide. Don't spend more than 5 min on this.

**Comment by roj1512** [comment]
We can also talk about the private translators group and this bot in the Translation Guide, but I personally think it is not needed because saying that translators will have “push access” does it all.

**Review by KnorpelSenf — APPROVED** [review]
Thanks!

**Pull request merged**